ui_framework/doc_site/src/views/table/table_with_menu_buttons.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import {
+  KuiTable,
+  KuiTableRow,
+  KuiTableRowCell,
+  KuiTableHeaderCell,
+} from '../../../../components';
+
+import {
+  RIGHT_ALIGNMENT
+} from '../../../../services';
+
+export function TableWithMenuButtons() {
+  return (
+     <KuiTable>
```

Comment:
Nitpick: looks like lines 16 and 17 were indented with triple spaces? Does this happen elsewhere?